eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -44,6 +44,8 @@ function Stage-Apiview-File($packagePath, $reviewId, $revisionId)
     $tokenFilePath = Join-Path $packagePath "cadl-output"
     $stagingReviewPath = Join-Path $OutputDir $reviewId
     $stagingPath = Join-Path $stagingReviewPath $revisionId
+    Write-Host "Out files in cadl-output"
+    Get-ChildItem -Path $tokenFilePath

```

Comment:
Do you want recursion on this?

Reply:
Nope. I only want to list first level output under `cadl-output` directory